### ISSUE TITLE
FreeHand tool does not work anymore !

### ISSUE THREAD

**Issue by MrMic** [kenwei/pdf_reader_x#221]
Since the last update, the FreeHand tool does not work anymore.

**Comment by bonillad** [comment]
I have this issue as well as the arrow annotation not showing up

**Comment by n3a1** [comment]
Yes. The freehand tool draws with click and drag but what you drew disappears as soon as you release the click.
